### DISCUSSION TITLE
How to auto-close file tree after open a file?

### DISCUSSION THREAD

**Discussion by omegahh** [nvim-tree/nvim-tree.lua#1710] [Q&A] [answered]
I don't find the option for this. Like `coc-explorer`. After I open a file, like pressing 'o', '<C-x>', or '<C-v>', The cursor focuses on the opened buffer, and file-tree is closed automatically.

**Comment by gegoune** [comment, marked as answer]
`:h nvim-tree.actions.open_file.quit_on_open`

**Reply by omegahh** [reply]
Thank you! I saw this before and tried once, but it didn't work. Now it seems like it is the packer's fault. I find that 'PackerCompile' not always work, sometimes it need to be run twice.
